lib/stytch/otps.rb

```diff
@@ -1,209 +1,592 @@
+      # An object with the following fields:
+      # request_id::
+      #   Globally unique UUID that is returned with every API call. This value is important to log for debugging purposes; we may ask for this value to help identify a specific API call when helping you debug an issue.
+      #   The type of this field is `String`.
+      # user_id::
+      #   The unique ID of the affected User.
+      #   The type of this field is `String`.
+      # phone_id::
+      #   The unique ID for the phone nunmber.
+      #   The type of this field is `String`.
+      # status_code::
+      #   The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+      #   The type of this field is `Integer`.
       def send(
         phone_number:,
         expiration_minutes: nil,
-        attributes: {},
+        attributes: nil,
+        locale: nil,
         user_id: nil,
         session_token: nil,
-        session_jwt: nil,
-        locale: nil
+        session_jwt: nil
       )
         request = {
-          phone_number: phone_number,
-          expiration_minutes: expiration_minutes
+          phone_number: phone_number
         }
-
-        request[:attributes] = attributes if attributes != {}
+        request[:expiration_minutes] = expiration_minutes unless expiration_minutes.nil?
+        request[:attributes] = attributes unless attributes.nil?
+        request[:locale] = locale unless locale.nil?
         request[:user_id] = user_id unless user_id.nil?
         request[:session_token] = session_token unless session_token.nil?
         request[:session_jwt] = session_jwt unless session_jwt.nil?
-        request[:locale] = locale unless locale.nil?
 
-        post_request("#{PATH}/send", request)
+        post_request('/v1/otps/whatsapp/send', request)
       end
 
+      # Send a one-time passcode (OTP) to a User's WhatsApp using their phone number. If the phone number is not associated with a User already, a User will be created.
+      #
+      # ### Next steps
+      #
+      # Collect the OTP which was delivered to the User. Call [Authenticate OTP](https://stytch.com/docs/api/authenticate-otp) using the OTP `code` along with the `phone_id` found in the response as the `method_id`.
+      #
+      # == Parameters:
+      # phone_number::
+      #   The phone number to use for one-time passcodes. The phone number should be in E.164 format. The phone number should be in E.164 format (i.e. +1XXXXXXXXXX). You may use +10000000000 to test this endpoint, see [Testing](https://stytch.com/docs/home#resources_testing) for more detail.
+      #   The type of this field is `String`.
+      # expiration_minutes::
+      #   Set the expiration for the Magic Link `token` in minutes. By default, it expires in 1 hour. The minimum expiration is 5 minutes and the maximum is 7 days (10080 mins).
```

Comment:
This should say OTP code not Magic Link `token`, I'll also update in docs same PR